### PR TITLE
Lesson 1 9

### PR DESCRIPTION
**Pull request by LiubovCass**
-Install React Router
-Setup Router
 Wrap existing JSX within new BrowserRouter component
 Inside BrowserRouter, wrap existing JSX within new Routes component
 Inside Routes, wrap existing JSX within new Route component with prop path equal to the root path ("/") and prop exact
note: the path prop will match any url that starts with the given string, so "/" will match all other urls ("/one", "/one/two", and so on) unless we specify that it must be an exact match using the exact prop
-Add New Route
 Below the Route component, create a new Route with path "/new"
 Inside the Route component, create a level-one heading with text "New Todo List"

### CODE REVIEW

File: src/pages/MyToDoApp.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+// }
+
+const API_ENDPOINT = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default/`;
+
+function MyToDoApp() {
+  const [todoList, setTodoList] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  React.useEffect(() => {
```
Comment (KingsleyIbe) [inline src/pages/MyToDoApp.js:29]: You could consider importing your state and effect hooks at the top e.g `import React, { useState, useEffect } from react`, by doing so you don't have to repeat `React.useState([]};` instead the following lines of code will look like this `const [todoList, setTodoList] = useState([]);` `const [isLoading, setIsLoading] = useState(true);` `useEffect(() => {}`

**Review by KingsleyIbe — APPROVED** [review]
Hello @LiubovCass ,

Your project is complete! There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉

Please check the inline optional suggestion(s).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/pages/MyToDoApp.js (hunk 1 above)
Comment (LiubovCass) [reply src/pages/MyToDoApp.js:29]: @KingsleyIbe Thank you for your review and suggestion. I definitely will use importing my state. Do I have to change my pull request now or can I do it in the next lessons?

Comment (KingsleyIbe) [reply src/pages/MyToDoApp.js:29]: You can effect these changes in the next update/lessons.

**Pull request merged**